File: src/main/java/io/github/takenoha/towerdefense/persistence/RewardDeliveryOutcome.java
package io.github.takenoha.towerdefense.persistence;

/** Result of reserving one durable reward queue row for a Paper inventory handoff. */
public enum RewardDeliveryOutcome {
    ACQUIRED,
    ALREADY_ACQUIRED,
    ALREADY_DELIVERED,
    HELD_BY_OTHER,
    VOIDED
}

File: src/main/kotlin/io/github/takenoha/towerdefense/persistence/RewardDeliveryOutcome.kt
package io.github.takenoha.towerdefense.persistence

/** Result of reserving one durable reward queue row for a Paper inventory handoff. */
enum class RewardDeliveryOutcome {
    ACQUIRED,
    ALREADY_ACQUIRED,
    ALREADY_DELIVERED,
    HELD_BY_OTHER,
    VOIDED,
}
